public/assets/sass/sass-components/home.scss

```diff
@@ -0,0 +1,4 @@
+.home-month-summary{
+    background-color: #4c4c4c3b;
```

Comment:
Consider adding variables, as this way the app theme would be inconsistent and every page might end up with a different color.